.clang-tidy

```diff
@@ -1,9 +1,9 @@
 ---
-Checks:          '*'
+Checks:          '*,-fuchsia-default-arguments'
```

Comment:
So this says that we don't want to enforce using default arguments for declarations or calls. Maybe we should reconsider keeping this check? Having this check would ensure that devs think about what values are used in a default state.

Reply:
Giving this more thought, I think this is a good check to remove. There are cases where having default arguments is helpful.

Reply:
Yes, the `fuchsia-default-arguments` was triggering this error (just on linux):

```
/home/travis/build/mapbox/node-cpp-skel/build/../src/object_async/hello_async.cpp:249:37: error: calling a function that uses a default argument is disallowed [fuchsia-default-arguments,-warnings-as-errors]
        return utils::CallbackError("first arg 'options' must be an object",
                                    ^
/usr/lib/gcc/x86_64-linux-gnu/4.9/../../../../include/c++/4.9/bits/basic_string.h:494:39: note: default parameter was declared here
      basic_string(const _CharT* __s, const _Alloc& __a = _Alloc());
```

https://travis-ci.org/mapbox/node-cpp-skel/jobs/418057304#L942. Disabling the `fuchsia-default-arguments` fixed the error, which I think is the best route because I can't see an easy way to workaround and I can imagine the usefulness of default args in downstream apps (even if we don't use them in node-cpp-skel).